### DISCUSSION TITLE
Force the default dashboard tab to "Following" tweak no longer an option?

### DISCUSSION THREAD

**Discussion by stellayurigasaki** [AprilSylph/XKit-Rewritten#1347] [Lounge]
Apologies if this has already been noticed but I recall that there was a tweak set in that forced the default dashboard tab to following. However, I do not see it anymore and there are random blogs that I am not following appearing on my dashboard. If a fix is being worked on, that would be greatly appreciated!

![2023-12-07 14_03_04-](https://github.com/AprilSylph/XKit-Rewritten/assets/142759416/b13f435a-e8f3-4d52-b4b2-69f7126dd612)

**Comment by marcustyphoon** [comment]
All blogs now have access to customizing their default dashboard tab, so if you have a newer blog you can switch to "following" without XKit! Click the settings icon all the way to the right of the tabs and "pin" the tab you want to be the default!

(naturally, if you have a ublock origin rule or user style/user script that completely removes the tab bar, you'll have to turn it off to do this; this is why I very specifically did not completely hide it in XKit.)

![image](https://github.com/AprilSylph/XKit-Rewritten/assets/8336245/9b34882f-dc26-44cb-ab50-8693eddebc01)

**Reply by stellayurigasaki** [reply]
Thank you very much! I was completely unaware of the fix tumblr implemented and I was wondering why random blogs were appearing on my dash.

Have a good day!!
